OpenRA.Mods.Common/Projectiles/AreaBeam.cs

```diff
@@ -21,6 +21,8 @@
 
 namespace OpenRA.Mods.Common.Projectiles
 {
+	public enum InaccuracyType { Maximum, PerCellIncrement, Absolute }
```

Comment:
This should then be moved to wherever the helper method goes (probably `OpenRA.Mods.Common/Util.cs`